ibis/backends/tests/test_generic.py

```diff
@@ -698,7 +707,12 @@ def test_bitwise_columns(backend, con, alltypes, df, op, left_fn, right_fn):
             lambda t: t.int_col,
             id="rshift_col_col",
         ),
-        param(rshift, lambda _: 3, lambda t: t.int_col, id="rshift_scalar_col"),
+        param(
+            rshift,
+            lambda _: 3,
+            lambda t: t.int_col,
+            id="rshift_scalar_col",
```

Comment:
This line looks the same as before, any reason to change the formatting?

Reply:
Restoring the formatting

Reply:
Restored.